examples/time_frequency/plot_epochs_spectra.py

```diff
@@ -33,6 +38,9 @@
 epochs = mne.Epochs(raw, events, event_id, tmin, tmax,
                     proj=True, baseline=(None, 0), preload=True,
                     reject=dict(grad=4000e-13, eog=150e-6))
+# Pull 2**n points to speed up computation
+epochs = EpochsArray(epochs.get_data()[..., :1024], epochs.info,
+                     epochs.events, tmin, epochs.event_id)
```

Comment:
It was a while ago - ok I'll give it a shot and see if it works.

On Thu, Jan 14, 2016 at 9:40 AM, Eric Larson <notifications@github.com>
wrote:

> In examples/time_frequency/plot_epochs_spectra.py
> <https://github.com/mne-tools/mne-python/pull/2710#discussion_r49758067>:
>
> > @@ -33,6 +38,9 @@
> >  epochs = mne.Epochs(raw, events, event_id, tmin, tmax,
> >                      proj=True, baseline=(None, 0), preload=True,
> >                      reject=dict(grad=4000e-13, eog=150e-6))
> > +# Pull 2**n points to speed up computation
> > +epochs = EpochsArray(epochs.get_data()[..., :1024], epochs.info,
> > +                     epochs.events, tmin, epochs.event_id)
>
> How long ago did you have that problem? We changed to using something that
> is a little bit more lenient with the floating point arithmetic that goes
> on under the hood about a year ago IIRC.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/2710/files#r49758067>.
>